apps/rollups/src/components/Layout.tsx

```diff
@@ -36,6 +36,28 @@ export const footerLinks = [
     },
 ];
 
+export const footerSupport = [
+    {
+        label: 'About Us',
+        href: 'https://cartesi.io/about/',
+    },
+    {
+        label: 'Docs',
+        href: 'https://docs.cartesi.io/',
+    },
+];
+
+export const footerGeneral = [
```

Comment:
@dandheedge, 
The links here are redundant. The Figma design is more of a reference in terms of where to go here.

Reply:
@brunomenezes alright. I will remove it from the Footer